desktop/analytics/partner_application.js

```diff
@@ -13,8 +13,17 @@ if (location.pathname.match('/gallery-partnerships')) {
   })
 
   $('#mktoForm_1238 .mktoButtonRow').click(function (e) {
+    window.email = $("#Email").val();
     analytics.track('Clicked bottom apply on gallery partnerships',
       {session_id: sd.SESSION_ID}
-    )
+    );
+    analytics.identify(
```

Comment:
I actually think we should pass email to the above event, not create a second one.